core-plugins/docs/Excel-batchsource.md

```diff
@@ -32,6 +32,12 @@ in **filePath** input. (Macro-enabled)
 **memoryTableName:** KeyValue table name to keep the track of processed files. This can be
 a new table or existing one. (Macro-enabled)
 
+**memoryDatasetExplorer:** Name of the Explorer table corresponding to 'memoryTableName' dataset. If not mentioned,
+name will be same as the 'memoryTableName' dataset. (Macro-enabled)
+
+**memoryDatasetExplorerDatabase:** Existing Database name for the 'memoryDatasetExplorer' dataset.
```

Comment:
Existing Hive database name for the 'memoryDatasetExplorer' dataset. By default, the table will be stored in the current namespace's database. (Macro-enabled)